static/templates/recent_topic_row.hbs

```diff
@@ -37,9 +37,15 @@
             </li>
             {{/if}}
             {{#each senders}}
-            <li class="recent_avatars_item" title="{{this.full_name}}">
-                <img src="{{this.avatar_url_small}}" class="recent_avatars_img" />
-            </li>
+                {{#if this.is_muted}}
+                <li class="recent_avatars_item" title="{{t 'Muted user'}}">
+                    <span><i class="fa fa-user recent_avatars_others"></i></span>
+                </li>
+                {{else}}
+                <li class="recent_avatars_item" title="{{this.full_name}}">
+                    <img src="{{this.avatar_url_small}}" class="recent_avatars_img" />
+                </li>
+                {{/if}}
```

Comment:
lgtm.  Two follow-up commit IDs come to mind:
* I think it'd be worth replacing the `title` attributes with TippyJS tooltips in a follow-up commit.
* Should we rename `recent_avatars_item` to `recent_topics_participant` for readability?

Reply:
- I think it would be better to show tooltips in the same position as the `title` (bottom right) because otherwise it will inevitable end up hiding some content in the same row.
- Yep!